src/azure-cli/azure/cli/command_modules/sql/commands.py

```diff
@@ -532,6 +534,21 @@ def load_command_table(self, _):
         g.command('list', 'list_by_server',
                   table_transformer=firewall_rule_table_format)
 
+    outbound_firewall_rules_operations = CliCommandType(
+        operations_tmpl='azure.mgmt.sql.operations#OutboundFirewallRulesOperations.{}',
+        client_factory=get_sql_outbound_firewall_rules_operations)
+
+    with self.command_group('sql server outbound-firewall-rule',
+                            outbound_firewall_rules_operations,
+                            client_factory=get_sql_outbound_firewall_rules_operations) as g:
+        g.custom_command('create', 'outbound_firewall_rule_create',
+                         table_transformer=outbound_firewall_rule_table_format)
+        g.command('delete', 'delete')
```

Comment:
what about add confirmation for delete command?

Reply:
We do not need the confirmation since it is one property itself. I am just following the procedure that we followed for Powershell APIs.